cmd/genji/shell/shell.go

```diff
@@ -72,6 +77,49 @@ func stdinFromTerminal() bool {
 	return true // data is from terminal
 }
 
+// RunInsertCommand Runs insert query from command line.
+func RunInsertCommand(e, DBPath, table string, data []string) error {
+	var ng engine.Engine
+	var err error
+
+	switch e {
+	case "bolt":
+		ng, err = boltengine.NewEngine(DBPath, 0660, nil)
+	case "badger":
+		ng, err = badgerengine.NewEngine(badger.DefaultOptions(DBPath).WithLogger(nil))
+	}
+
+	if err != nil {
+		return err
+	}
+
+	db, err := genji.New(ng)
+	if err != nil {
+		return err
+	}
+
+	defer db.Close()
+	if stdinFromTerminal() {
+		// Data that given to insert command.
+		for i := range data {
+			err := executeInsertCommand(db, table,[]byte(data[i]))
+			if err != nil {
+				return err
+			}
+		}
+
+		return nil
+	}
+
+	// Insert command is given in the pipe
+	in, err := ioutil.ReadAll(os.Stdin)
```

Comment:
We should not read all stdin because it can get very big, `os.Stdin` is an `io.Reader`, `json.NewDecoder` takes an `io.Reader`, we should simply pass it as is. This means that `executeInsertCommand` should expect a reader, not a byte slice

Reply:
I thought about it afterwards.